swagger_spec_validator/common.py

```diff
@@ -30,10 +36,18 @@ def validate_json(json_document, schema_path):
     schema_path = resource_filename('swagger_spec_validator', schema_path)
     with open(schema_path) as schema_file:
         schema = json.loads(schema_file.read())
-    resolver = RefResolver('file://{0}'.format(schema_path), schema)
+    resolver = RefResolver(
+        'file://{0}'.format(schema_path),
+        schema,
+        store={'http://swagger.io/v2/schema.json': schema})
```

Comment:
But the key `id` is not always present in the schema (it is present in 2.0 schema but not in `api_declaration` and `resource_listing` schema of 1.2).

Also, from signature of `RefResolver`, `store` takes a `dict` (`id` would be a `str`).